src/main/webapp/app/tasks/task.service.ts

```diff
@@ -1,32 +1,57 @@
-import { Observable } from 'rxjs';
+import { Observable } from 'rxjs'
 
-import { Task } from 'app/tasks/task';
+import { Task } from 'app/tasks/task'
 
 /**
  * Service interface for implementations that handle tiny tasks.
  */
 export interface TaskService {
-
   /**
    * Returns the list of all tasks.
    *
    * @returns an `Observable` holding the list of tasks
    */
-  getAll(): Observable<Task[]>;
+  getAll(): Observable<Task[]>
 
   /**
    * Adds a new task to the list of tasks.
    *
    * @param name the task's name
    * @returns an `Observable` holding the created task
    */
-  create(name: string): Observable<Task>;
+  create(name: string): Observable<Task>
 
   /**
    * Removes the task with the given ID from the list of tasks.
    *
    * @param id the ID of the task to be removed
    * @returns an empty `Observable`
    */
-  delete(id: string): Observable<void>;
+  delete(id: string): Observable<void>
+
+  /**
+   * Mark the task as done.
+   *
+   * @param id the ID of which is done
+   * @returns an empty `Observable`
+   */
+  setIsDone(status: string): Observable<void>
```

Comment:
in `setIsDone` of local-task.service you use the parameter as id 🤔  must be rather `setIsDone(id: string): Observable<void>`